src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -71,7 +71,7 @@ static License GetTrialLicense()
             }
             else
             {
-                var message = $"Trial for Particular Service Platform is still active, trial expires on {trialLicense.ExpirationDate.Value.ToLocalTime().ToShortDateString()}.";
+                var message = $"Trial for the Particular Service Platform has been active since {trialStartDate.ToLocalTime().ToShortDateString()}.";
```

Comment:
```
{trialStartDate.ToLocalTime().ToShortDateString()}
```

Shouldn't that be the following so that the string interpolation takes care of formatting the data object and not process an already formatted string?

```
{trialStartDate.ToLocalTime():d}
```